calendar-vue/src/components/calendar/month-view/CalendarMonthDay.vue

```diff
@@ -42,9 +42,10 @@ export default {
       const selected = this.selectedCheck();
       const today = this.todayCheck();
 
-      const firstOrLast = this.isFirstOrLast ? 'first-or-last' : '';
+      const shouldBeGreyedOut = this.shouldBeGreyedOut ?
```

Comment:
`shouldBeGreyedOut` -> `greyedOut` as this is a string for the class rather than a boolean.